tools/cdk-build-tools/config/eslintrc.js

```diff
@@ -32,13 +32,16 @@ module.exports = {
     // Require use of the `import { foo } from 'bar';` form instead of `import foo = require('bar');`
     '@typescript-eslint/no-require-imports': [ 'error' ],
     '@typescript-eslint/indent': [ 'error', 2 ],
+
+    // Style
     'quotes': [ 'error', 'single', { avoidEscape: true } ],
+    'comma-dangle': [ 'error', 'always-multiline' ], // ensures clean diffs
```

Comment:
can you add the link to the medium article here as well?

Reply:
done, can you reapprove?